docker-compose.yml

```diff
@@ -47,7 +47,7 @@ services:
       - ./:/opt/src
     links:
       - api
-    command: bash -c "locust -f /opt/src/locustfile.py --host=http://api:8001 --no-web -c 250 -r 25 --run-time 1m"
+    command: bash -c "locust -f /opt/src/locustfile.py --host=http://api:8001 --headless --users 250 --hatch-rate 25 --run-time 30s1m"
```

Comment:
kan 30s1m ook als 1m30s geschreven worden? Leest makkelijker

Reply:
This was a mistake. It should have just been 1m. I fixed it in the PR below.